openwisp_notifications/settings.py

```diff
@@ -3,6 +3,16 @@
 
 CONFIG_DEFAULTS.update({'USE_JSONFIELD': True})
 
+OPENWISP_NOTIFICATION_EMAIL_TEMPLATE = getattr(
+    settings,
+    'OPENWISP_NOTIFICATION_EMAIL_TEMPLATE',
+    'openwisp_notifications/email_template.html',
+)
+
+OPENWISP_NOTIFICATION_EMAIL_LOGO = getattr(
+    settings, 'OPENWISP_NOTIFICATION_EMAIL_LOGO', 'https://git.io/JfwB3'
```

Comment:
please make a note (eg: an issue) to update this once we merge it, so that we can change the default URL to point to the github URL.

Reply:
This already points to the location on master branch:
https://raw.githubusercontent.com/openwisp/openwisp-notifications/master/openwisp_notifications/static/openwisp_notifications/images/openwisp-logo.png
I used git,io to shorten URL.

Reply:
@pandafy let's not shorten it please, users may have the impression we're hiding something. Please also add a point to the README which advises to change this setting to point to their own web server and not to github.